seqr/views/utils/json_utils.py

```diff
@@ -36,13 +37,15 @@ def render_with_initial_json(html_page, initial_json):
         indent=4,
         default=DjangoJSONEncoderWithSets().default
     )
-
     html = loader.render_to_string(html_page)
 
     html = html.replace(
         "window.initialJSON=null",
         "window.initialJSON="+initial_json_str
     )
+
+    import pdb; pdb.set_trace()
```

Comment:
haven't use this before. would be cool to see how you use it.

Reply:
it sets a breakpoint and drops you in a python interpreter at that point. Should definitly not be committed though